### DISCUSSION TITLE
Bounces

### DISCUSSION THREAD

**Discussion by lTaySoNl** [postalserver/postal#2443] [Help with using Postal]
How can I get a list of mailboxes that receive an error report when sending them? I want to clear the address database of unused or non-existent email addresses.

**Comment by willpower232** [comment]
Postal doesn't keep this information long term so doesn't generate reports. You may want to implement webhooks in your application https://docs.postalserver.io/developer/webhooks so that you can do it all automatically.

**Reply by lTaySoNl** [reply]
Please help me set up Postal so that I receive bounce messages in MailWizz. I'm a beginner and I don't understand how to do it.

**Reply by sinnbeck** [reply]
Doesn't that come with a built in mailserver? If not, you need to find the docs that explains how to use an external mailserver. I assume it will tell you how to set up a webhook handler

**Reply by kayastha1988** [reply]
This is how my webhooks is that I have created in postal. How the bounce records data from postal pass the data to the define URL for webhooks. I have attached image below.

Looking forward to hearing from you.

Thanks.

![image](https://github.com/postalserver/postal/assets/19544006/de1c1401-6980-493b-b494-baced02a0b8b)
